src/utils/graph.ts

```diff
@@ -303,6 +281,71 @@ export function formatGraphResponse<E extends EntityKey>(
   return []
 }
 
+export async function querySubgraph<
```

Comment:
**praise**: returning a promise makes using this fn a lot nicer - sweet improvement 💯